agent/api/task/task_windows.go

```diff
@@ -43,6 +43,10 @@ const (
 	cpuSharesPerCore  = 1024
 	percentageFactor  = 100
 	minimumCPUPercent = 1
+
+	// windowsDefaultNetworkName is the name of the docker network to which the containers in
+	// default mode are attached to.
+	windowsDefaultNetworkName = "nat"
```

Comment:
Minor: Do you need to prefix the variable with "windows" here, since this file is already Windows-specific?